main.scss

```diff
@@ -0,0 +1,159 @@
+@import 'variables';
+* {
+    box-sizing: border-box;
+    font-family: raleway, sans-serif;
+}
+
+.container {
+    width: 100%;
+}
+
+
+/* Hero Image etc */
+
+.hero {
+    position: relative;
+    // idea for fully scaling background-img comes from
+    // https://www.smashingmagazine.com/2013/07/simple-responsive-images-with-css-background-images/
+    // TODO: implement <img srcset> and create different sized images 
+    // http://www.responsivebreakpoints.com/
+    #hamburger {
+        display: none;
+    }
+    #banner {
+        width: 100%;
+        display: inline-block;
+        vertical-align: middle;
+        font: 0/0 serif;
+        text-shadow: none;
+        color: transparent;
+        background-size: 100%;
+        background-position: 50% 50%;
+        background-repeat: no-repeat;
+        background-image: url('./img/header-2.jpg');
```

Comment:
Since this has a different header image now, should we remove the old one from the PR? Or another idea... could the images rotate on each page / reload or something?